website/docs/r/vapp_firewall_rules.html.markdown

```diff
@@ -0,0 +1,146 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vapp_firewall_rules"
+sidebar_current: "docs-vcd-resource-vapp-firewall-rules"
+description: |-
+  Provides a vCloud Director vApp Firewall resource. This can be used to create, modify, and delete firewall settings and rules.
+---
+
+# vcd\_vapp\_firewall\_rules
+
+Provides a vCloud Director vApp Firewall resource. This can be used to create,
+modify, and delete firewall settings and rules in a [vApp network](/docs/providers/vcd/r/vapp_network.html).
```

Comment:
Could we add a sentence on which vApp network types the FW can be created and on which not? Currently, you have to find out the hard way that the isolated network doesn't support it:
```
vcd_vapp_firewall_rules.vapp_fw: Creating...

Error: error setting firewall rules: provided network isn't connecd to org network or isn't fenced

  on yorktown.tf line 359, in resource "vcd_vapp_firewall_rules" "vapp_fw":
 359: resource "vcd_vapp_firewall_rules" "vapp_fw" {
```

Reply:
added

Reply:
Word "to" missing:

> Firewall rules can be applied to networks connected **to** Org network or vApp networks which are fenced.

Reply:
added